cmd/vela-terraform/apply.go

```diff
@@ -123,23 +123,20 @@ func (a *Apply) Command(dir string) *exec.Cmd {
 
 	// check if Vars is provided
 	if len(a.Vars) > 0 {
-		var vars string
 		for _, v := range a.Vars {
-			vars += fmt.Sprintf("-var=%s ", v)
+			flag := fmt.Sprintf(`-var=%s`, v)
+			// add flag for Vars from provided command
+			flags = append(flags, flag)
```

Comment:
Do you think we could condense this into one line?

```suggestion
			// add flag for Vars from provided command
			flags = append(flags, fmt.Sprintf(`-var=%s`, v))
```